src/components/Footer.js

```diff
@@ -1,9 +1,17 @@
+import Link from 'next/link';
+
 export default function Footer() {
 	return (
 		<>
-			<button type="button">My Contact</button>
-			<button type="button">Add contact</button>
-			<button type="button">Contact list</button>
+			<Link type="button" href="/profile/my-contact">
+				<button type="button">My Contact</button>
```

Comment:
Please do not use button inside a Link or inside an a. You should use span or a-tag (just for next-Link)